syntax="proto2";
package onething;

option cc_generic_services = true;

message AsrRequest {
    optional string type = 2;
};

message AsrResponse {
    required int32 code = 1;
    optional string msg = 2;
};

service AsrProxyService {
    rpc asr(AsrRequest) returns (AsrResponse);
};